helm/chart/teresa/requirements.yaml

```diff
@@ -1,5 +1,5 @@
-dependencies:
-- name: minio
-  version: "2.5.11"
-  repository: "https://kubernetes-charts.storage.googleapis.com/"
-  condition: useMinio
+#dependencies:
+#- name: minio
+#  version: "2.5.11"
+#  repository: "https://kubernetes-charts.storage.googleapis.com/"
+#  condition: useMinio
```

Comment:
Will we need this file? what about remove it?

Reply:
In my opinion, we can remove this file. I think that minio can be a outsourced resource, like S3, GCS.

Reply:
Hm, mixed feelings, I like the idea of there's a way to with one command get Teresa up and running in a dev environment. 

But, maybe we never back here 😝 

IMO, there's no problem if you remove this file.

Reply:
Teresa accepts any kind of S3-like storage. I guess the minio dependency only exists for development ease, but it can be provisioned standalone.

Reply:
> Will we need this file? what about remove it?

File removed !